scripts/configure_gcc.mk

```diff
@@ -4,7 +4,7 @@
 # debug build?
 DEBUG = 1
 # optimization
-OPT = -Og -fsingle-precision-constant -O3 
+OPT = -O1 -fsingle-precision-constant #-O3 
```

Comment:
Will need to do with optimization. You'll want to make sure the code still works with optimization on. Sometimes it affects the flags between interrupt contexts

Reply:
Reverted optimization settings.